src/ShopListFilterBundle/objects/db/TPkgShopListfilterItemList.class.php

```diff
@@ -119,6 +120,9 @@ protected function &_NewElement(&$aData)
             $sTableObject = $oListfilterItemType->fieldClass;
         }
 
+        /**
+         * @FIXME Empty if-statement?
```

Comment:
This statement should be safe to remove (unless we are depending on the side effect of the autoloader kicking in when calling `class_exists` - and I hope we are not depending on that.)